packages/synapse-react-client/src/components/IconSvgButton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { IconButton, Tooltip } from '@mui/material'
+import IconSvg, { IconName } from './IconSvg/IconSvg'
+
+export type IconSvgButtonProps = {
+  icon: IconName
+  tooltipText: string
+  onClick?: (e: React.MouseEvent) => void
+  disabled?: boolean
+  href?: string
+}
+export function IconSvgButton(props: IconSvgButtonProps) {
+  const { icon, tooltipText, onClick, disabled, href } = props
+  const iconSvg = <IconSvg icon={icon} wrap={false} />
+  return (
+    <Tooltip key={tooltipText} title={tooltipText} placement={'top'}>
+      <span>
+        {href ? (
```

Comment:
Nick, help me with the types!  I tried so many things to try to bring these into a single IconButton.  
For example, tried 
href={'href' in props ? props.href : undefined}
but this is a type error from IconButton (ButtonBase states href is required?)